internal/pkg/template/jsonnet/function/function.go

```diff
@@ -124,6 +126,18 @@ func InstanceIDShort(instanceIDShort string) *jsonnet.NativeFunction {
 	}
 }
 
+// GeneratorInstanceIDShort Jsonnet function returns a random, shortened id of the template instance.
+func GeneratorInstanceIDShort() *jsonnet.NativeFunction {
+	return &jsonnet.NativeFunction{
+		Name:   `GeneratorInstanceIDShort`,
+		Params: ast.Identifiers{},
+		Func: func(params []any) (any, error) {
+			instanceID := idgenerator.TemplateInstanceID()
```

Comment:
Is this random enough? Like some UUID?

Reply:
I don't understand the question. I just used id generator which was used before.

Reply:
I have checked it, it should be safe. Asked you question but I had to go though the implementation by myself